frontend/mrs.developer.json

```diff
@@ -136,6 +136,6 @@
     "url": "git@github.com:kitconcept/kitconcept.solr.git",
     "https": "https://github.com/kitconcept/kitconcept.solr.git",
     "path": "frontend",
-    "branch": "2.0.0-alpha.2"
+    "branch": "2.0.0-alpha.3"
```

Comment:
@sneridagh @ericof @reekitconcept FYI: This might work, but it should really be `tag` instead of `branch`